messaginginapp/src/main/java/io/customer/messaginginapp/ModuleMessagingInApp.kt

```diff
@@ -97,4 +57,52 @@ class ModuleMessagingInApp(
                 ?: throw IllegalStateException("ModuleMessagingInApp not initialized")
         }
     }
+
+    override fun embedMessage(message: Message, elementId: String) {
+        moduleConfig.eventListener?.messageShown(InAppMessage.getFromGistMessage(message))
```

Comment:
That's one place where using `get()` would be a good choice in my opinion 😅 

```kotlin
private val eventListener: InAppEventListener?
    get() = moduleConfig.eventListener
```

Reply:
I am not sure, how? `ModuleMessagingInApp` can't be created without `MessagingInAppModuleConfig` which takes `eventListener`. 

There's no way to modify the `eventListener` after ModuleMessagingInApp is initialized, so what benefit are we getting from using `get()` here?

Reply:
Discussed on call, this isn't needed at the moment